meson.build

```diff
@@ -15,6 +15,14 @@ project(
 	meson_version: '>= 0.59.0'
 )
 
+# see https://github.com/wwmm/easyeffects/pull/1739 for more info
+cxx = meson.get_compiler('cpp')
+cxx_version=cxx.version()
+if cxx.get_id() == 'clang'
+  clang_version = cxx_version.version_compare('>=16.0.0')
+  assert(clang_version == true, 'This project only supports gcc or > clang 16 compilers due to usage of c++20 features.')
```

Comment:
You're asserting `true == true` or potentially `false == true`, for the record, and both evaluate to the value of the first (non-constant) argument. You could just as easily error with:

```meson
assert(clang_version, 'This project only ........')
```

But you could also do this:
```meson
if cxx.get_id() == 'clang' and not cxx_version.version_compare('>=16.0.0')
    error('This project only ................')

Reply:
Thanks @eli-schwartz. I have updated the master branch with your suggestion.

Reply:
Oops, if you are going to drop cxx_version = cxx.version() you also need to inline that. Which to be fair, it does make sense to inline it.

So, `and not cxx_version.version_compare('>=16.0.0')` -> `and not cxx.version().version_compare('>=16.0.0')`

Reply:
Of course! That is what happens when we just look at the code without actually testing it :smile:. Done!